asserts/preseed.go

```diff
@@ -173,10 +173,12 @@ func checkPreseedSnaps(snapList interface{}) ([]*PreseedSnap, error) {
 		}
 		seen[preseedSnap.Name] = true
 		snapID := preseedSnap.SnapID
-		if underName := seenIDs[snapID]; underName != "" {
-			return nil, fmt.Errorf("cannot specify the same snap id %q multiple times, specified for snaps %q and %q", snapID, underName, preseedSnap.Name)
+		if snapID != "" {
+			if underName := seenIDs[snapID]; underName != "" {
```

Comment:
Wouldn't `if underName, ok := seenIDs[snapID]; ok {` work as well? Then we wouldn't need the empty string check

Reply:
Oh sorry, you're completely right. I misunderstood